pkg/crc/cluster/cluster.go

```diff
@@ -223,3 +223,15 @@ func DeleteOpenshiftApiServerPods(oc oc.OcConfig) error {
 
 	return errors.RetryAfter(60, deleteOpenshiftApiserverPods, time.Second)
 }
+
+func CheckProxySettingsForOperator(oc oc.OcConfig, proxy *network.ProxyConfig, deployment, namespace string) (bool, error) {
+	cmdArgs := []string{"set", "env", "deployment", deployment, "--list", "-n", namespace}
+	out, _, err := oc.RunOcCommandPrivate(cmdArgs...)
```

Comment:
why `Private`?

Reply:
@cfergeau as part of output it list the proxy details.